pf/tests/internal/testprovider/cmd/pulumi-resource-testbridge/schema.json

```diff
@@ -48,12 +48,31 @@
                 },
                 "bic": {
                     "type": "boolean"
+                },
+                "phases": {
+                    "$ref": "#/types/testbridge:index/TestnestRuleActionParametersPhases:TestnestRuleActionParametersPhases"
                 }
             },
             "type": "object",
             "required": [
                 "automaticHttpsRewrites",
-                "bic"
+                "bic",
+                "phases"
+            ]
+        },
+        "testbridge:index/TestnestRuleActionParametersPhases:TestnestRuleActionParametersPhases": {
+            "properties": {
+                "p1": {
+                    "type": "boolean"
+                },
+                "p2": {
+                    "type": "boolean"
+                }
+            },
+            "type": "object",
+            "required": [
+                "p1",
+                "p2"
```

Comment:
These should be optional. I'm not 100% sure if this is an existing bug or if this change introduced a new bug. My guess is existing.

Reply:
I've got a fix, one sec.